pa11y-config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "chromeLaunchConfig": {
+      "args": ["--no-sandbox", "--disable-dev-shm-usage"]
```

Comment:
We wouldn't want to use `--no-sandbox` for a production environment (if you were running pa11y with our live website), but it's required to have it work with GitHub Actions.